### ISSUE TITLE
Combining alt and captions for images in the PhotoPack component for screen readers

### ISSUE THREAD

**Issue by deaxmachina** [reuters-graphics/graphics-components#57]
I have a question, and a suggestion re accessibility in the PhotoPack component, though this might apply to other components / patterns. I'd like to preface this by saying I'm very new to both screen reader testing and to writing screen reader compatible code, so I might be missing the point here and would appreciate any tips. 

What I noticed when I used the mac VO on the PhotoPack component is that it will **first read out the alt text for each image, and after that** the captions for the images in sequence. Is there anything in the current structure of the code the suggest that these should be paired? In particular, I noticed an ` aria-labelledby` attribute linking each image with its caption, but I don't know if it does anything? 
Here is the code corresponding to that: 

For the image itself: 
```
<figure aria-labelledby="{id}-figure-{ri}-{i}">
    <img
        src="{img.src}"
        alt="{img.altText}"
    />
</figure>
```

For the caption: 
```
<div id="{id}-figure-{ri}-{i}" class="caption">
    {@html marked(img.caption)}
</div>
```

If I'm misunderstanding what this is supposed to do, or not using the screen reader correctly, can someone please clarify? 

Otherwise, **I'd like to propose the following alternative**:
- Put the alt text and caption together like so: 
```
<figure aria-labelledby="{id}-figure-{ri}-{i}">
    <img
        src="{img.src}"
        alt="{`${img.altText} Caption: ${img.caption}`}"
    />
</figure>
```
For a video and not an image (I've got a separate PR to include videos in this component) this would look like so: 
```
<div class="visually-hidden">
    <p>{`${img.altText} Caption: ${img.caption}`}</p>
</div>
```

- Hide the container with all the captions from screen reader. 
```
<div class="captions-container" aria-hidden="true">
    // captions code
</div>
```

If I can get any thoughts on this, that'd be great. Tagging everyone I think might be relevant: @MinamiFunakoshiTR @hobbes7878 @juruwolfe @pkd2512
